src/common/tagging/code2cloud/tag_group.go

```diff
@@ -16,15 +18,33 @@ func (t *TagGroup) InitTagGroup(_ string, skippedTags []string, explicitlySpecif
 	}
 	t.SkippedTags = skippedTags
 	t.SpecifiedTags = explicitlySpecifiedTags
-	t.SetTags([]tags.ITag{&YorTraceTag{}})
+	t.SetTags([]tags.ITag{&YorTraceTag{}, &YorNameTag{}})
 }
 
 func (t *TagGroup) GetDefaultTags() []tags.ITag {
 	return []tags.ITag{
 		&YorTraceTag{},
+		&YorNameTag{},
 	}
 }
 
 func (t *TagGroup) CreateTagsForBlock(block structure.IBlock) error {
-	return t.UpdateBlockTags(block, struct{}{})
+	return t.UpdateBlockTags(block)
+}
+
+func (t *TagGroup) UpdateBlockTags(block structure.IBlock) error {
```

Comment:
@rotemavni Why is this needed? What were we missing?

Reply:
@nimrodkor This method is implemented in the base block, but doesn't pass the `block` into `tag.CalculateValue(block)`. We need to pass it on so we can determine the block name for the tag.